backend/services/regions.csv

```diff
@@ -0,0 +1,38 @@
+Name,Country,Latitude,Longitude
+Abia,Nigeria,5.454095299,7.5153071
+Adamawa,Nigeria,9.512977199,12.3881887
+Akwa Ibom,Nigeria,4.9408638,7.8412267
+Anambra,Nigeria,6.218313599,6.9531842
+Bauchi,Nigeria,10.6228284,10.0287754
+Bayelsa,Nigeria,4.7629786,6.028898
+Benue,Nigeria,7.350574699,8.7772877
+Borno,Nigeria,12.1875392,13.3080034
+Cross River,Nigeria,5.867196599,8.5204774
+Delta,Nigeria,5.527306099,6.1784167
+Ebonyi,Nigeria,6.199691799,8.0348906
+Edo,Nigeria,6.607657499,5.9722713
+Ekiti,Nigeria,7.736890999,5.2738326
+Enugu,Nigeria,6.553609399,7.4143061
+Federal Capital Territory,Nigeria,8.831122799,7.1724673
+Gombe,Nigeria,10.38301,11.206567
+Imo,Nigeria,5.585945599,7.0669651
+Jigawa,Nigeria,12.3252362,9.5103296
+Kaduna,Nigeria,10.3825318,7.8533226
+Kano,Nigeria,11.8948389,8.5364136
+Katsina,Nigeria,12.5630825,7.6207063
+Kebbi,Nigeria,11.4167574,4.1074545
+Kogi,Nigeria,7.794960199,6.6868669
+Kwara,Nigeria,8.836789099,4.6688487
+Lagos,Nigeria,6.526903299,3.5774005
+Nasarawa,Nigeria,8.438786799,8.2382849
+Niger,Nigeria,9.932608299,5.6511088
+Ogun,Nigeria,6.978858199,3.4389293
+Ondo,Nigeria,7.020968599,5.0567477
+Osun,Nigeria,7.548404699,4.4978307
+Oyo,Nigeria,8.215124899,3.5642897
+Plateau,Nigeria,9.058344599,9.6826289
+Rivers,Nigeria,4.8416028,6.8604088
+Sokoto,Nigeria,13.0611195,5.3152203
+Taraba,Nigeria,8.014133399,10.7376336
+Yobe,Nigeria,12.1233242,11.5065937
+Zamfara,Nigeria,12.0078998,6.4191432
```

Comment:
This is where we can have the Python script to define the query to get all regions with admin_level=4 in country=x (eg. Nigeria)